beacon-chain/blockchain/service.go

```diff
@@ -153,6 +153,9 @@ func (c *ChainService) ProcessBlock(block *types.Block) error {
 		return nil
 	}
 	if canProcess {
+		if err := c.chain.processAttestations(block); err != nil {
```

Comment:
This branch is untested. Can we add a test to ensure the attestations are processed on a block when canProcess is truthy?

Reply:
Done. Added `TestProcessingBlockWithAttestations` in blockchain's `service_test.go`